packages/babel-core/cjs-proxy.cjs

```diff
@@ -19,7 +19,13 @@ const functionNames = [
   "transformFromAst",
   "parse",
 ];
-const propertyNames = ["types", "tokTypes", "traverse", "template"];
+const propertyNames = [
+  "buildExternalHelpers",
```

Comment:
This is used by out test runner, which needs to load Babel 8 to run plugins with it.